recipes/libaatm/meta.yaml

```diff
@@ -0,0 +1,40 @@
+
+{% set version = '1.0.1' %}
+{% set sha256 = 'da5ff1f700396ade1fd8781d24ca8e5c92cba40a45b040cf278cf62e996b173d' %}
+
+{% set build = 0 %}
+
+package:
+  name: libaatm
+  version: {{ version }}
+
+source:
+  fn: libaatm-{{ version }}.tar.gz
+  url: https://github.com/hpc4cmb/libaatm/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+  run:  
+
```

Comment:
you need to add the ``test`` section
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=test#test